### ISSUE TITLE
walmart-ca Navigation timeout of 30000 ms exceeded when using proxy

### ISSUE THREAD

**Issue by PMVX** [jef/streetmerchant#2071]
## Expected Behavior

When using proxies, it should continue to work like any other stores.

## Current Behavior

Whenever a proxy is used, either in dotenv, global.proxies or walmart-ca.proxies, I get the error: 
Navigation timeout of 30000 ms exceeded

## Steps to Reproduce

I have tried global proxies, store.proxies and I noticed that the only time I get a response from walmart-ca query is when I'm not using a proxy. I have tried using a proxy that rotates every call and tried a rotating proxy that changes every few minutes as well. 
Putting that same proxy in internet explorer allows me to manually visit the website with no issues. I can confirm that the proxy works because other stores work fine and it also shows a different location whenever I visit whatismyip.com

## Environment

- OS: Win10 running in HyperV

```
AUTO_ADD_TO_CART=false
IN_STOCK_WAIT_TIME=30
OPEN_BROWSER=false
PROXY_ADDRESS=xxx.xxx.xxx.xxx
PROXY_PROTOCOL=http
PROXY_PORT=12345
SHOW_ONLY_SERIES=xboxsx,sonyps5de,sonyps5c,sonyps5cplus,sonyps5cspidey,sonyps5deplus,xboxss
STORES=walmart-ca,canadacomputers
USER_AGENT="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.190 Safari/537.36"
```

## Logs

[2:07:54 AM] info :: ℹ puppeteer config:  {
  "0": "--proxy-server=http://xxx.xxx.xxx.xxx:xxxxx"
}
[2:08:07 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[2:08:16 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[2:08:30 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[2:08:30 AM] error :: ✖ [1/16] [walmart-ca] sony sonyps5c ps5 console - Navigation timeout of 30000 ms exceeded
[2:08:41 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[2:09:00 AM] error :: ✖ [2/16] [walmart-ca] sony sonyps5de ps5 digital - Navigation timeout of 30000 ms exceeded
[2:09:09 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[2:09:18 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[2:09:30 AM] error :: ✖ [3/16] [walmart-ca] microsoft xboxsx xbox series x - Navigation timeout of 30000 ms exceeded
[2:09:30 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[2:09:39 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK

**Comment by PMVX** [comment]
I have also created a new folder and started a clean install of streetmerchant and copying my current dotenv and lowering the page timeout - I get the same result. If I disable the proxy and connect to a VPN, it works just fine.

ＳＴＲＥＥＴＭＥＲＣＨＡＮＴ -- on rotating proxy
[10:47:33 AM] info :: ℹ selected series: sonyps5de, sonyps5c, xboxss, xboxsx
[10:47:34 AM] info :: ℹ puppeteer config:  {
  "0": "--proxy-server=http://xxx.xxx.xxx.xxx:xxxxx"
}
[10:47:41 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:47:45 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:47:49 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:47:50 AM] error :: ✖ [walmart-ca] sony sonyps5c ps5 console - Navigation timeout of 15000 ms exceeded
[10:47:52 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:47:55 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:47:58 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:48:02 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:48:05 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:48:05 AM] error :: ✖ [walmart-ca] sony sonyps5de ps5 digital - Navigation timeout of 15000 ms exceeded
[10:48:08 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:48:12 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:48:15 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:48:17 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:48:20 AM] error :: ✖ [walmart-ca] microsoft xboxsx xbox series x - Navigation timeout of 15000 ms exceeded
[10:48:21 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:48:26 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:48:30 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:48:35 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:48:35 AM] error :: ✖ [walmart-ca] microsoft xboxss xbox series s - Navigation timeout of 15000 ms exceeded
[10:48:40 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK

Commented out the 3 proxy settings in dotenv and connected to a VPN:
ＳＴＲＥＥＴＭＥＲＣＨＡＮＴ
3.5.0
[10:56:04 AM] info :: ℹ selected stores: walmart-ca, canadacomputers
[10:56:04 AM] info :: ℹ selected series: sonyps5de, sonyps5c, xboxss, xboxsx
[10:56:20 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:21 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:21 AM] error :: ✖ [walmart-ca] sony sonyps5c ps5 console - Navigation timeout of 15000 ms exceeded
[10:56:23 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:24 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:25 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:27 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:27 AM] info :: ✖ [walmart-ca] [sony (sonyps5de)] ps5 digital :: OUT OF STOCK
[10:56:28 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:29 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:31 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:32 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:33 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:35 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:36 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:37 AM] info :: ✖ [walmart-ca] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:38 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:39 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:40 AM] info :: ✖ [walmart-ca] [microsoft (xboxss)] xbox series s :: OUT OF STOCK
[10:56:41 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:42 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:44 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:44 AM] info :: ✖ [walmart-ca] [sony (sonyps5c)] ps5 console :: OUT OF STOCK
[10:56:46 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:48 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK
[10:56:48 AM] info :: ✖ [walmart-ca] [sony (sonyps5de)] ps5 digital :: OUT OF STOCK
[10:56:50 AM] info :: ✖ [canadacomputers] [microsoft (xboxsx)] xbox series x :: OUT OF STOCK

**Comment by PMVX** [comment]
I've set timeout now to 60 and from time to time I see a walmart response.
I ran it with headless=false and noticed that those navigation timeout issue corresponds to walmart displaying a page about "Are you Human? 
![image](https://user-images.githubusercontent.com/79793641/109828505-11d00380-7c13-11eb-8b65-4e8dc911ceb2.png)

I also frequently see the walmart product page opening and then closing after a few seconds but nothing is displayed on the console. After several minutes I get an error 444
![image](https://user-images.githubusercontent.com/79793641/109829779-3c6e8c00-7c14-11eb-90ee-6a3f9f2c1e5e.png)
